build_resources.yml

```diff
@@ -0,0 +1,71 @@
+# YAML Configuration File for Qt Resource Building
+#
+# This YAML file defines the configuration for building Qt UI and resource files using the tk-build-qt-resources script.
+# Each entry in the file represents a separate build configuration, to know more about each field please visit
+# https://github.com/shotgunsoftware/tk-toolchain?tab=readme-ov-file#tk-build-qt-resources.
+
+- ui_src: resources
+  res_files:
+    - resources
+  py_dest: python/tk_multi_demo/ui
+  import_pattern: tank.platform.qt
```

Comment:
Remove default values if https://github.com/shotgunsoftware/tk-toolchain/pull/65#discussion_r1698774692 is approved